tools/es5to6-files-per-user.js

```diff
@@ -18,37 +18,37 @@ const filesPerHuman = (filesToCheck, humans) =>
     Math.max(filesToCheck.length / humans.length);
 
 const commercial = [
-    'janua',
-    'JonNorman',
-    'regiskuckaertz',
-    'rich-nguyen',
-    'katebee',
-    'Calum-Campbell',
-    'ScottPainterGNM',
+    'francis.carr@guardian.co.uk',
+    'jon.norman@guardian.co.uk',
+    'regis.kuckaertz@guardian.co.uk',
+    'rich.nguyen@guardian.co.uk',
+    'kate.whalen@guardian.co.uk',
+    'calum.campbell@guardian.co.uk',
```

Comment:
@Calum-Campbell  is not part of the commercial team anymore, but @lps88 is. Also I'm not sure @ScottPainterGNM will be comfortable handling these. Scott, happy to pair with you if you want.